pkg/apis/build/v1alpha2/cluster_stack_conversion.go

```diff
@@ -2,20 +2,34 @@ package v1alpha2
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
this is very small but can you put the external library import `corev1 "k8s.io/api/core/v1"` with the other external libraries?

```
"context"
"encoding/json"
"fmt"

corev1 "k8s.io/api/core/v1"
"knative.dev/pkg/apis"

Reply:
Thanks for your comment! Please see 6708a3fa07133220e954fd14099525f284497720.